caput/pipeline.py

```diff
@@ -1263,7 +1266,13 @@ def read_process_write(self, input, input_filename, output_filename):
             output_dirname = os.path.dirname(output_filename)
             if not os.path.isdir(output_dirname):
                 os.makedirs(output_dirname)
-            self.write_output(output_filename, output)
+            self.write_output(
```

Comment:
`write_output()` lost its `self` parameter (It is now a `staticmethod`, so I am not quite sure how this has been working for my tests!